src/common/POLFields/FieldHolding/FieldHolding.js

```diff
@@ -94,27 +88,32 @@ const FieldHolding = ({
     />
   );
 
-  if (isHoldingsLoading) return <Loading />;
+  if (isLoading) return <Loading />;
+
+  const locationForNewHolding = () => {
+    if (!selectedLocation) return null;
+
+    return isDisabled
+      ? (
+        <KeyValue
+          label={labelId ? <FormattedMessage id={locationLabelId} /> : ''}
+          value={`${selectedLocation.name}(${selectedLocation.code})`}
+        />
+      )
+      : (
+        <TextField
+          label={labelId ? <FormattedMessage id={locationLabelId} /> : ''}
+          required={required}
+          disabled
+          value={`${selectedLocation.name}(${selectedLocation.code})`}
+          endControl={clearButton}
+        />
+      );
```

Comment:
AcqTextField has this `isNonInteractive` functionality

Reply:
Right. One thing - location is UUID, but user wants to see location name)